### DISCUSSION TITLE
Extract bold or italic words and phrases from PDF text document

### DISCUSSION THREAD

**Discussion by J0hn3ch** [py-pdf/pypdf#2057] [Q&A]
Hi!

Is there any way to retrieve from a text extraction in pypdf, those words or phrases that are visualized in bold or italic? 
Also for different headers content

Thanks!

**Comment by pubpub-zz** [comment]
you should have a look at the documentation:
https://pypdf.readthedocs.io/en/stable/user/extract-text.html#using-a-visitor
